data-transformers/transformers/mysqlTransformer.py

```diff
@@ -123,17 +165,25 @@ def main():
                     indexes[schema[i].replace('"', '')] = i
         
         # Uses Spark to map lines to Cassandra queries
-        f1 = lambda a: createDic(a, trialID, experimentID, conf, types, schema, indexes)
-        f2 = lambda a: filterMock(a, limit)
-        query = data.map(lambda line: line.decode().split(",")).filter(f2).map(f1)
+        query = data.map(lambda line: line.decode().split(",")) \
+                    .filter(lambda a: filterMock(a, limit)) \
+                    .map(lambda a: createDic(a, trialID, experimentID, conf, types, schema, indexes))
+        
+        query = sc.parallelize(query.take(2000))
+        
+        cutProcesses = []
+        if conf["dest_table"] == "process":
+            query = cutNInitialProcesses(query, nProcessesToIgnore)
+            cutProcesses = query.filter(lambda a: a["to_ignore"] is True).map(lambda a: a["process_instance_id"]).collect()
+            print cutProcesses
+        elif conf["dest_table"] == "construct":
+            query = cutConstructs(query, cutProcesses)
```

Comment:
How are you sure you first compute `conf["dest_table"] == "process"`? This would be the only case in which you actually have `cutProcesses` with the correct value.